bin/cli.py

```diff
@@ -11,15 +14,24 @@ def main():
     argv = sys.argv
     if len(argv) < 2 or argv[1] == "help":
         BaseCommand().help()
-        return False
+        return
 
     command = get_command_class(argv)
 
-    if command is not False:
+    if command:
         instance = command()
-        return instance.handle()
-    BaseCommand().help()
-    return False
+        try:
+            instance.sdk.login()
+            return instance.handle()
```

Comment:
what is the purpose of returning the instance.handle() here? Looking at the command classes we don't return anything.

Reply:
No point other than to exit the function could might as well be `instance.handle()
return` the `return instance.handle()` was just the way it was set up beforehand. Didn't give it any thought

Reply:
Since we don't return anything from the commands: do a return at the end and do sys.exit(main()) when calling main()
Then we can support return values to sys.exit() in the future